pilot/pkg/model/config_test.go

```diff
@@ -582,7 +591,10 @@ func TestAuthenticationPolicyConfig(t *testing.T) {
 			},
 			PrincipalBinding: authn.PrincipalBinding_USE_ORIGIN,
 		},
+		"mockopenidonce":  makeGoldenPolicyForMockOpenIdDiscovery("mockopenidonce"),
```

Comment:
This is not golden (expected output policy). It's just regular policy input with empty jwksuri. Basically, this map provides an easy way to seed config store for test, and verify later (with the previous assumption Get return the same value as input). As you are now changing the return policy, may be you want to change the test structure (e.g in the test cases struct below, put the actual golden proto in "expected", instead of a string to refer to an entry in this map; you can get rid of this map as well).

Reply:
moved test to auth_cache_test.go and reverted change on this file.